### PR TITLE
update glider popups, add uframe glider tracks

### PR DESCRIPTION
**Pull request by birdage**
@DanielJMaher see below

### CODE REVIEW

**Comment by birdage** [comment]
screen shot 2015-09-29 at 10 42 05 am

**Comment by DanielJMaher** [comment]
It only goes 3.6 meters deep?  that sounds wrong.  you sure you populating it right?  @birdage

**Comment by birdage** [comment]
@DanielJMaher yup the services will get it from uframe directly, unless their interpolation `limit` function,  is not working right on depth

**Comment by birdage** [comment]
@DanielJMaher the services route fix, wont come in to effect until i update the services

**Comment by DanielJMaher** [comment]
the extra comments make this hard to understand... are you saying this is ready for review?

**Comment by DanielJMaher** [comment]
@FBRTMaka Can you review and merge?>

**Comment by birdage** [comment]
@DanielJMaher updated the label to `ready for review`

**Comment by FBRTMaka** [comment]
@birdage @DanielJMaher viewing now.

File: ooiui/static/js/views/common/MapView.js [hunk 1]
```diff
@@ -179,7 +188,42 @@ var MapView = Backbone.View.extend({
     self.gliderCollection.each(function(model,i) {      
       var gliderTrackLayer = L.geoJson(model.toJSON(), {style: gliderTrackStyle});
       //popup
-      var popupContent = '<p><strong>' + model.get('name') + '</strong><br>';
+
+      var glName     = model.get('name').split('-')[1].substring(2)
```
Comment (FBRTMaka) [inline ooiui/static/js/views/common/MapView.js]: Semicolon.

File: ooiui/static/js/views/common/MapView.js [hunk 2]
```diff
@@ -179,7 +188,42 @@ var MapView = Backbone.View.extend({
...
+      var glLocation = model.get('name').split('-')[0].substring(0,2);      
+      var gliderText = "Glider " + glName
```
Comment (FBRTMaka) [inline ooiui/static/js/views/common/MapView.js]: Semicolon

File: ooiui/static/js/views/common/MapView.js [hunk 3]
```diff
@@ -179,7 +188,42 @@ var MapView = Backbone.View.extend({
     self.gliderCollection.each(function(model,i) {      
       var gliderTrackLayer = L.geoJson(model.toJSON(), {style: gliderTrackStyle});
       //popup
-      var popupContent = '<p><strong>' + model.get('name') + '</strong><br>';
+
+      var glName     = model.get('name').split('-')[1].substring(2)
+      var glLocation = model.get('name').split('-')[0].substring(0,2);      
+      var gliderText = "Glider " + glName  
+      
+      var glPosition = model.get('coordinates').slice(-1).pop()
```
Comment (FBRTMaka) [inline ooiui/static/js/views/common/MapView.js]: semicolon...

File: ooiui/static/js/views/common/MapView.js [hunk 4]
```diff
@@ -39,6 +39,15 @@ var MapView = Backbone.View.extend({
...
                         }
 
+    self.arrayCodes =   {
+                         "CP":"Coastal Pioneer",
+                         "CE":"Endurance & Cabled Array",
+                         "GP":"Station Papa",
+                         "GI":"Irminger Sea",
+                         "GA":"Argentine Basin",
+                         "GS":"Southern Ocean"
+                        }
```
Comment (FBRTMaka) [inline ooiui/static/js/views/common/MapView.js]: semicolon

File: ooiui/core/routes/common.py [hunk 5]
```diff
@@ -325,7 +325,7 @@ def get_toc_list():
 @app.route('/api/uframe/glider_tracks', methods=['GET'])
 def get_glider_track():  
     token = get_login()
-    response = requests.get(app.config['SERVICES_URL'] + '/uframe/get_glider_track/'+request.args['id'], auth=(token, ''), data=request.args)
+    response = requests.get(app.config['SERVICES_URL'] + '/uframe/get_glider_tracks', auth=(token, ''), data=request.args)
```
Comment (FBRTMaka) [inline ooiui/core/routes/common.py]: semicolon

Comment (birdage) [inline ooiui/core/routes/common.py]: semicolon in python?

**Comment by FBRTMaka** [comment]
:+1:  nice @birdage !

**Pull request merged**